### PR TITLE
Pop up window

### PR DESCRIPTION
**Pull request by tatouroberto**
## What does this PR do?
1. Store all data from in javascript array
2. Adding Window popup on portfolios Projects

## How can this changes be manually tested
1. Clone this project
6. Navigate into cloned Project
7. run `npm install`
8. open index.html in browser

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Popup functionality in place ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] While I think you have done a good job, I think you should work more on the transparency of the app.
- [ ] I would recommend you work on the alignment of the text and buttons I pointed out in the image, you will notice there is space between that of the Figma.
### Your Design
image

### Figma
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -494,13 +508,91 @@ hr {
...
+  width: 100vw;
+  height: 100vh;
+  background-color: #c1c7d0;
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  backdrop-filter: blur(5px);
+  opacity: 0.8;
+  overflow: auto;
+}
```
Comment (Alusp) [inline style.css:521]: - [ ] Good job in implementing the change requested by the previous Reviewer, :clap: However, as requested by the previous reviewer, your background texts is affecting your popup cards, Please resolve it :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

Popup Student


</td>
      <td>

Design Popup


</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @tatouroberto, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional PR Summary Flow used ✔️
- [ ] Popup closes and opens Successfully :heavy_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] Please ensure that you add at least 2 `Future Features` in your readme file. :slightly_smiling_face: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @tatouroberto in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bienvenuushindi — APPROVED** [review]
# Approved 👏  

## Hi @tatouroberto   👋, 

Well done implementing the requested changes 👏

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @bienvenuushindi  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**